artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueManagerImpl.java

```diff
@@ -72,7 +72,7 @@ public static void performAutoDeleteQueue(ActiveMQServer server, Queue queue) {
       ActiveMQServerLogger.LOGGER.autoRemoveQueue("" + queue.getName(), queue.getID(), "" + queue.getAddress());
 
       try {
-         server.destroyQueue(queueName, null, true, false, settings.isAutoDeleteAddresses(), true);
+         server.destroyQueue(queueName, null, true, false, false, true);
```

Comment:
This almost certainly rendered the settings unused, so they should have been removed as well.